test/Mono.Linker.Tests.Cases/Mono.Linker.Tests.Cases.csproj

```diff
@@ -114,6 +114,7 @@
     <Content Include="LinkXml\UnusedTypeDeclarationPreservedByLinkXmlIsKept.xml" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\src\linker\Mono.Linker.csproj" />
```

Comment:
This does not make much sense. You need to add the reference to https://github.com/mono/linker/pull/976/files#diff-38e7184cf84a72cb81b88450ae657f74R7 not here